app/views/home/_closed_warm_up.html.haml

```diff
@@ -28,3 +28,6 @@
         %p.text-center
           Make sure you don't miss out -
           = link_to "register to vote now!", "https://gov.uk/register-to-vote"
+
+
+= render partial: "news_sign_up"
```

Comment:
I don't think we need this one?  This page already has the actionnetwork partial on it.

Reply:
Fixed